style.css

```diff
@@ -0,0 +1,912 @@
+  #opening {
+    display: grid;
+    grid-template-columns: 1fr 10fr;
+    grid-template-areas: 'social intro_word';
+    height: 870px;
+    background-position: 100% 110%;
+    background-image: url("./media/illustration-header.svg");
+  }
+
+  #headline-container {
+    margin-top: 290px;
+  }
+
+  #intro_word {
+    grid-area: intro_word;
+    margin-top: 80px;
+  }
+
+  #recent-img {
+    width: 654px;
+    height: 474px;
+  }
+
+  #about_section {
+    background-image: url("./media/illustration-about-me.svg");
+    background-position: 100% 25%;
+  }
+
+  #contact > p {
+    margin-left: 100px;
+  }
+
+  #form {
+    margin-right: 100px;
+  }
+
+  #social {
+    grid-area: social;
+    margin-top: 180px;
+    margin-left: 0;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  #description {
+    max-width: 566px;
+  }
+
+  #about {
+    max-width: 566px;
+  }
+
+  #resume-button {
+    margin-top: 0;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  #logo {
+    display: block;
+    float: left;
+  }
+
+  #mess {
+    display: none;
+  }
+
+  #bars {
+    display: block;
+    float: right;
+  }
+
+  .the-nav1 {
+    color: #fff5e1;
+    padding: 20px;
+  }
+
+  .the-nav2 {
+    color: #fff5e1;
+    padding: 20px;
+  }
+
+  #opening {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-template-areas: 'intro_word' 'social';
+    padding-left: 24px;
+  }
+
+  #intro_word {
+    grid-area: intro_word;
+    margin-top: 50px;
+  }
+
+  #social {
+    grid-area: social;
+    display: flex;
+    flex-direction: row;
+    margin-top: 20px;
+    padding: 0;
+    align-items: flex-start;
+  }
+
+  #bottom_socials {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  #bottom-social {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 25px 0;
+  }
+
+  .pop-menu {
+    width: auto;
+    height: 500px;
+    background: #3c3a39;
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  .pop {
+    display: none;
+  }
+
+  #close {
+    float: right;
+    margin-top: 10px;
+    margin-right: 20px;
+    width: 20px;
+    height: 20px;
+    background: #3c3a39;
+    border: none;
+  }
+
+  .img-button {
+    width: 100%;
+    height: 100%;
+  }
+
+  .nav-tools {
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    color: #fff5e1;
+    list-style: none;
+    width: 270px;
+    margin-top: 30px;
+    border-bottom: 1px solid #ccc;
+    overflow-y: hidden;
+  }
+
+  .first-tool {
+    padding-top: 20px;
+  }
+
+  li a:link {
+    text-decoration: none;
+  }
+
+  .navigation {
+    background: #3c3a39;
+    padding: 20px;
+  }
+
+  .the-nav3 {
+    font-family: 'Roboto', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 4px;
+    color: #fff5e1;
+    padding-left: 20px;
+  }
+
+  .space {
+    height: 550px;
+    background: #3c3a39;
+  }
```

Comment:
- [ ] Good job by making some of the changes required by the previous reviewer. But as he mentioned to match your design to the Figma template, so, I noticed that you added some unnecessary code lines that make your mobile menu not match the mobile menu Figma template. 
  - I can suggest first you remove the empty `<div class="space"></div>` in the `index.html` file and its styling code in the `style.css` file as it makes the mobile menu be like two blocks(with a horizontal line just after the navigation options separating them). 
  - Then add the  `height` property in the `.navigation{ ... }` class with a value of something like this: `.navigation{ height: 100vh; }`(or a bit greater than **100vh**) to fill the whole screen, also remember to **increase the width of the `border-bottom`** of each navigation option and correctly **fix the `CloseMenu` button at his expected position as it so close to the top-right corner.** 

| Figma Design | Your Design |
| ------------------- | ------------------- |
| ![Screenshot from 2022-07-18 23-33-09](https://user-images.githubusercontent.com/83042742/181114361-9b96ed7b-93ba-4a13-a192-4d12ee61e6b4.png) |  ![Screenshot from 2022-07-26 23-16-42](https://user-images.githubusercontent.com/83042742/181114494-a4d68186-e28b-4b8f-ba2a-cc3704b1467e.png) |